azure/postgresql/terraform/outputs.tf

```diff
@@ -0,0 +1,16 @@
+output "server_name" {
+  value = "${var.postgres_name}"
```

Comment:
nit: 
Perhaps this file doesn't need the interpolation syntax?
Possible improvement: `value = var.postgres_name`